lib/resty/upstream/healthcheck.lua

```diff
@@ -230,11 +230,13 @@ local function check_peer(ctx, id, peer, is_backup)
         ok, err = sock:connect(name)
     end
     if not ok then
-        return peer_error(ctx, is_backup, id, peer, "failed to connect to ", name, ": ", err)
```

Comment:
I used a tool suggested at https://github.com/openresty/lua-resty-upstream-healthcheck/pull/36#discussion_r100677253 and it complained about line length, therefore broke it.